src/main/java/com/yahoo/sketches/hash/MurmurHash3.java

```diff
@@ -107,7 +107,7 @@ private MurmurHash3() {}
     int rem = ints - tail; // remainder ints: 0,1,2,3
     
     // Get the tail
-    long k1 = 0;
+    long k1;
```

Comment:
Well, you got it half right.  Both k1 and k2 should be uninitialized here, and then k2 = 0 added to the else clause.  This eliminates any unnecessary assignment.  Fixed in master.